pkg/controller/gkenetworkparamset/gkenetworkparamset_controller.go

```diff
@@ -83,6 +90,10 @@ func NewGKENetworkParamSetController(
 		networkInformerFactory:   networkInformerFactory,
 	}
 
+	c.nodeLister = nodeInformer.Lister()
+	c.nodeInformerSynced = nodeInformer.Informer().HasSynced
```

Comment:
Couldn't you have kept the same structure and initialized nodeLister and nodeInformerSynced with the other fields?

Reply:
Done.